src/package/VSIXProject/TestPlatform.csproj

```diff
@@ -102,66 +102,19 @@
       <VSIXSubPath>Extensions\CppUnitFramework</VSIXSubPath>
     </VsixSourceItem>
 
-    <VsixSourceItem Include="$(VsixInputFileLocation)\TestHost\ComComponents\x64\*.*">
-      <VSIXSubPath>TestHost\ComComponents\x64</VSIXSubPath>
+    <VsixSourceItem Include="$(VsixInputFileLocation)\Extensions\TestImpact\x64\*.*">
```

Comment:
@acesiddhu @singhsarab need your inputs on this. Given testhost is flattened now, will the msdia in ..\x64 suffice?

Reply:
For TestImpact these four lines are needed 
```
    <VsixSourceItem Include="$(VsixInputFileLocation)\Extensions\TestImpact\ComComponents\x64\*.*">
      <VSIXSubPath>Extensions\TestImpact\ComComponents\x64</VSIXSubPath>
    </VsixSourceItem>
    <VsixSourceItem Include="$(VsixInputFileLocation)\Extensions\TestImpact\ComComponents\x86\*.*">
      <VSIXSubPath>Extensions\TestImpact\ComComponents\x86</VSIXSubPath>
    </VsixSourceItem>
```